srcpkgs/evince/template

```diff
@@ -21,7 +21,7 @@ homepage="https://wiki.gnome.org/Apps/Evince"
 #changelog="https://gitlab.gnome.org/GNOME/evince/-/raw/main/NEWS"
 changelog="https://gitlab.gnome.org/GNOME/evince/-/raw/gnome-44/NEWS"
```

Comment:
```suggestion
changelog="https://gitlab.gnome.org/GNOME/evince/-/raw/gnome-${version%.*}/NEWS"
```

Reply:
no, let's keep hardcoding the version in the template, it's much easier to just click on the link instead of having to use xbps-src show to get it

Reply:
ok, left it hardcoded.